api-gateway/src/test/java/com/petclinic/bffapigateway/presentationlayer/ApiGatewayControllerTest.java

```diff
@@ -827,7 +827,27 @@ public void getBillById(){
 
     }
 
+    @Test
+    public void getBillByOwnerId(){
```

Comment:
[nitpick] the test name should be plural since it returns a flux of bills -> getBillsByOwnerId.

Reply:
changed to bills